### PR TITLE
feat(Icons): Added missing lock icon

### PR DESCRIPTION
**Pull request by priyankar205**

### CODE REVIEW

**Comment by codepretty** [comment]
The lock with keyhole is a deprecated design for Teams theme. Please use the icons-lock from the ProcessedIcons folder.

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `black` and `white` options for the `color` prop of the `Label` component @mnajdova ([#855](https://github.com/stardust-ui/react/pull/855))
 - Add `Flex` component @kuzhelov ([#802](https://github.com/stardust-ui/react/pull/802))
 - Add `inline` prop for `Dropdown` component @Bugaa92 ([#863](https://github.com/stardust-ui/react/pull/863))
+- Export `lock-new` SVG icon @priyankar205 ([#872](https://github.com/stardust-ui/react/pull/872))
```
Comment (kuzhelov) [inline CHANGELOG.md:54]: would like @codepretty to take a look on this PR before merge

**Review by kuzhelov — CHANGES_REQUESTED** [review]
is waiting for resolution from design team

**Comment by alinais** [comment]
@priyankar205 , @codepretty I believe that this PR doesn't make sense after the merge of #906, right?

**Pull request closed without merging**